pinot-perf/pom.xml

```diff
@@ -87,6 +87,15 @@
       <groupId>org.testng</groupId>
       <artifactId>testng</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Do we use testng or mockito in perf? If so, we should probably change them to `compile` scope

Reply:
The `testng` dependency was already there so not too sure about that one and I tried running a few benchmarks after removing it and they work fine. I can go ahead and remove it in this PR if you'd like.

 Mockito is needed for `BenchmarkQueries` and `BenchmarkOrderByQueries` because they extend `BaseQueriesTest` which does use Mockito - https://github.com/apache/pinot/blob/64eb3237d782a2b5dd092edae2415b8c1d12bf37/pinot-core/src/test/java/org/apache/pinot/queries/BaseQueriesTest.java#L78

I've updated the scope for `mockito-core` to `compile`.